spa_ui/self_service/gulpfile.js

```diff
@@ -62,7 +64,7 @@ gulp.task('autotest', task('test', {singleRun: false}));
 /**
  * Serves up injected html for dev, builds for evything else.
  */
-gulp.task('serve-dev', ['dev-fonts', 'dev-images', 'dev-skin-images', 'dev-imgs', 'inject'], task('serve', {
+gulp.task('serve-dev', ['dev-fonts', 'dev-images', 'dev-skin-images', 'dev-imgs', 'gettext-compile', 'inject'], task('serve', {
```

Comment:
I think extracting & compiling is something we should do only now and then (before devel freeze for example).